sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormat.scala

```diff
@@ -156,7 +156,7 @@ trait FileFormat {
    * Returns whether this format supports the given [[DataType]] in read/write path.
    * By default all data types are supported.
    */
-  def supportDataType(dataType: DataType, isReadPath: Boolean): Boolean = true
+  def supportsDataType(dataType: DataType): Boolean = true
```

Comment:
Sorry, but we also have `supportBatch` in this file. If this is just a cosmetic issue, I prefer to keep consistency in the single class instead of other classes. Also, we still use `supportCodegen` in other classes, too.
> 2. rename to supportsDataType, which is more consistent with other data source API(e.g. SupportsBatchRead, SupportsBatchWrite).

Reply:
+1